src/sts.rs

```diff
@@ -0,0 +1,278 @@
+//! The AWS STS API.
+//!
+//! If you're using the service, you're probably looking for [StsClient](struct.StsClient.html).
+#![allow(unused_imports)]
```

Comment:
Yes, this shouldn't be there and won't cause any harm to clean up.
I have a hunch that there are some combinations of features that cause things to be imported but then not used and then trip an unused import error. I suppose we'll fix that as and when.